workers/unity/Packages/com.improbable.gdk.core/Editor/SingletonScriptableObject.cs

```diff
@@ -4,9 +4,9 @@
 using UnityEditor;
 using UnityEngine;
 
-namespace Improbable.Gdk.BuildSystem.Configuration
+namespace Improbable.Gdk.Core.Editor
 {
-    internal abstract class SingletonScriptableObject<TSelf> : ScriptableObject
+    public abstract class SingletonScriptableObject<TSelf> : ScriptableObject
```

Comment:
Docstrings?

Reply:
Yes! Actually all of this needs docstrings